pkg/destroy/aws/aws.go

```diff
@@ -73,32 +78,34 @@ func (o *ClusterUninstaller) Run() error {
 
 	awsConfig := &aws.Config{Region: aws.String(o.Region)}
 
-	// Relying on appropriate AWS ENV vars (eg AWS_PROFILE, AWS_ACCESS_KEY_ID, etc)
-	awsSession, err := session.NewSession(awsConfig)
-	if err != nil {
-		return err
+	if o.Session == nil {
+		// Relying on appropriate AWS ENV vars (eg AWS_PROFILE, AWS_ACCESS_KEY_ID, etc)
+		o.Session, err = session.NewSession(awsConfig)
+		if err != nil {
+			return err
+		}
+		o.Session.Handlers.Build.PushBackNamed(request.NamedHandler{
```

Comment:
With this inside the `if o.Session == nil` block, the installer no longer adds the user-agent when destroying.

Reply:
> With this inside the if o.Session == nil block, the installer no longer adds the user-agent when destroying.

Fixed with 72319c7f4 -> bb4396613.